page.go

```diff
@@ -53,6 +54,19 @@ func (p *page) meta() *meta {
 	return (*meta)(unsafeAdd(unsafe.Pointer(p), unsafe.Sizeof(*p)))
 }
 
+func (p *page) fastCheck(id pgid) {
+	if p.id != id {
+		log.Panicf("Page expected to be: %v, but self identifies as %v", id, p.id)
```

Comment:
Can we just panic? Usually bbolt is supposed to be embedded the client application, and we either return an error or panic directly. It might not be good to write a log to stderr.

Reply:
Done. Used _assert_.